From a4a31e73897b51b73b1be994d36759584407914f Mon Sep 17 00:00:00 2001
From: Han Xu <b45815@freescale.com>
Date: Fri, 9 Oct 2015 10:16:24 -0500
Subject: [PATCH] MLK-11719-1: imx: config: enable debugfs in mfg_defconfig

enable debugfs in mfg_defconfig for mfgtool to read bch layout setting
from debugfs.

Signed-off-by: Han Xu <b45815@freescale.com>
---
 arch/arm/configs/imx_v7_mfg_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/imx_v7_mfg_defconfig b/arch/arm/configs/imx_v7_mfg_defconfig
index 3093eec..2a88710 100644
--- a/arch/arm/configs/imx_v7_mfg_defconfig
+++ b/arch/arm/configs/imx_v7_mfg_defconfig
@@ -302,6 +302,7 @@ CONFIG_NLS_ASCII=y
 CONFIG_NLS_ISO8859_1=y
 CONFIG_NLS_ISO8859_15=m
 CONFIG_NLS_UTF8=y
+CONFIG_DEBUG_FS=y
 CONFIG_MAGIC_SYSRQ=y
 # CONFIG_SCHED_DEBUG is not set
 # CONFIG_DEBUG_BUGVERBOSE is not set
-- 
1.8.0

